rebasehelper/cli.py

```diff
@@ -164,6 +178,9 @@ def run():
         except RebaseHelperError as e:
             logger.error('\n%s', six.text_type(e))
             sys.exit(1)
+        except EnableOptionFormatException as e:
+            sys.stderr.write("{}".format(e))
+            sys.exit(1)
```

Comment:
Please remove these three lines, as the new exception is not needed. Also in the future please be consistent with the way we log things. Just have a look at the lines above your addition.